### PR TITLE
[INT][LF-963] date-select class `is-disabled` ignored in calendar

### PR DESCRIPTION
**Pull request by RipripKipkip**
image

### CODE REVIEW

**Comment by sanlucca** [comment]
Fix or Update for 4.2 ? If it's a fix, it should go to master

File: packages/ng/libraries/date/src/lib/calendar/calendar-input.component.scss [hunk 1]
```diff
@@ -151,6 +151,13 @@
 			}
 		}
 	}
+
+	&.is-disabled {
+		pointer-events: none;
+		&:hover .calendar-grid-item-content {
```
Comment (lucienbertin) [inline packages/ng/libraries/date/src/lib/calendar/calendar-input.component.scss:157]: with `pointer-events: none`, it will trigger the `:hover` ?

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1049

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1049

**Pull request merged**